src/common.jl

```diff
@@ -0,0 +1,12 @@
+
+"From http://www.pserc.cornell.edu/matpower/MATPOWER-manual.pdf Table B-4"
+@enum GeneratorCostModel begin
+    piecewise_linear = 1
+    polynomial = 2
+end
```

Comment:
Should the attribute names here be all upper case? I don't think it's an issue if it is the way it is right now, since they end up as constants in the module scope and cannot be redefined.

```
julia> piecewise_linear = 2
ERROR: invalid redefinition of constant piecewise_linear
Stacktrace:
 [1] top-level scope at none:0

julia>
```

I think there's an argument to be made for the names of the enum to be upper case, since they are Instances of a struct that are `const` in the module scope.

Reply:
I meant to revisit this after reading JuMP.  These are essentially constants in PowerSystems.  The JuMP style guide says this:

@enum SomeEnum ENUM_VALUE_A ENUM_VALUE_B

I think this rule is best.

Reply:
Fixed

Reply:
Resolved in f7b5427